### DISCUSSION TITLE
does ALT+[ and ALT+] and CTRL+Backspace can remap in Neovim?

### DISCUSSION THREAD

**Discussion by jacquen** [community/community#7247] [Copilot Conversations] [answered]
Hi, Tpope:

I installed copilot.nvim on my nvim, seems change suggestion (ALT+] and ALT+[) and open copilot (CTRL+Backspace) had some conflict with existing key map, is possible remap those keys in Copilot? Thanks.

**Comment by tpope** [comment, marked as answer]
These keys are not currently mapped by copilot.vim.

**Reply by diegograssino** [reply]
Old but actual ... wich keys can i remap? Im having problems remapping as the docs says (doesnt work), and make me choice between COC and Copilot .... and both are my best friends!
